doc/USERMANAGEMENT.md

```diff
@@ -173,20 +171,181 @@ Now you can go to `import`, paste that template and click `proceed` and the user
 
 #### Add the user manually
 
-You can add a user manually through phpLDAPadmin, after connecting go to `⭐️Create new entry here` :
+You can add a user manually through phpLDAPadmin, after connecting go to `Create new entry here` :
 
 <img src="images/installation/Create_new.gif" alt="Create user"/>
 
 You can for example create a user in the default admin group `cn=admin,dc=ldap,dc=cetic,dc=be`, or create a new group in which you can create new users. 
 
-In this example we are going to create a simple user under the default admin user (which is also a group).
+In this example, we are going to create a simple user under the default admin user (which is also a group).
```

Comment:
Not sure I understand what "under" does mean?
Why mention that admin is a group here?

Reply:
The default user ( admin ) can be a group in which we can add users, this is the case where you don't need to divide users into multiple groups.

Reply:
ok¸please clarify the sentence to reflect this info.